src/number_theory/pell.lean

```diff
@@ -90,6 +89,7 @@ def xz (n : ℕ) : ℤ := xn n
 /-- The Pell `y` sequence, considered as an integer sequence.-/
 def yz (n : ℕ) : ℤ := yn n
 /-- The element `a` such that `d = a ^ 2 - 1`, considered as an integer.-/
+@[nolint unused_arguments]
```

Comment:
Can you add a comment to justify this?

Reply:
In fact, it was unjustified, so I've removed it.